### PR TITLE
DOC-8001: Docs for Enhance the metrics experience for CockroachDB Serverless

### PR DESCRIPTION
**Pull request by florence-crl**
Addresses DOC-8001

(1) Updated cloud Metrics page, added Custom Metrics Chart section and removed link to Available metrics. (2) Added cloud Custom Metrics Chart page which includes metric-names-serverless.md and ui-custom-metrics-chart-page.html and has images. (3) Added metric-names-serverless.md based on cockroachlabs/managed-service PR 14438. (4) Added ui-custom-metrics-chart-page.html based on v23.1/ui-custom-chart-debug-page-00.html. (5) Updated Cloud Deployments sidebar, added Custom Metrics Chart Page link. (6) Patched Preview include. (7) Added cloud release note.

### CODE REVIEW

**Review by kevin-v-ngo — COMMENTED** [review]
Submitting an initial review for now

File: src/current/cockroachcloud/custom-chart-page.md
Comment (florence-crl) [reply src/current/cockroachcloud/custom-chart-page.md:7]: updated

File: src/current/cockroachcloud/custom-chart-page.md
Comment (florence-crl) [reply src/current/cockroachcloud/custom-chart-page.md:9]: updated

File: src/current/cockroachcloud/custom-chart-page.md
Comment (florence-crl) [reply src/current/cockroachcloud/custom-chart-page.md:2]: updated

File: src/current/_includes/cockroachcloud/metric-names-serverless.md
Comment (kevin-v-ngo) [inline src/current/_includes/cockroachcloud/metric-names-serverless.md:61]: We removed these metrics for V1. @THardy98 FYI

File: src/current/cockroachcloud/custom-metric-chart-page.md
Comment (kevin-v-ngo) [inline src/current/cockroachcloud/custom-metric-chart-page.md:17]: Consistency Nit: Is time 'span' in our docs? Always thought it was time range (e.g., "time range").

File: src/current/cockroachcloud/custom-metric-chart-page.md
Comment (kevin-v-ngo) [inline src/current/cockroachcloud/custom-metric-chart-page.md:34]: What are the default values of the Preview? We should call it out here. @THardy98

Comment (kevin-v-ngo) [inline src/current/cockroachcloud/custom-metric-chart-page.md:34]: What are the default values of the Preview? We should call it out here. @THardy98

File: src/current/cockroachcloud/custom-metric-chart-page.md
Comment (kevin-v-ngo) [inline src/current/cockroachcloud/custom-metric-chart-page.md:30]: What is the retention period and interval of metrics? We should call it out. @THardy98

**Review by kevin-v-ngo — COMMENTED** [review]
Great Florence!

Looks good to me. Approved pending my comments and Eng sign-off from Thomas.

File: src/current/_includes/cockroachcloud/metric-names-serverless.md
Comment (florence-crl) [reply src/current/_includes/cockroachcloud/metric-names-serverless.md:61]: Deleted commented lines.

File: src/current/cockroachcloud/custom-metric-chart-page.md
Comment (florence-crl) [reply src/current/cockroachcloud/custom-metric-chart-page.md:17]: updated. You are correct: docs use time range or time interval.

File: src/current/cockroachcloud/custom-metric-chart-page.md
Comment (florence-crl) [reply src/current/cockroachcloud/custom-metric-chart-page.md:34]: The only default in the Preview is that it uses the page's selected time interval. Updated with this information.

Comment (florence-crl) [reply src/current/cockroachcloud/custom-metric-chart-page.md:34]: same as above

**Review by florence-crl — COMMENTED** [review]
Updated as suggested. Thanks for the review!

File: src/current/_includes/cockroachcloud/metric-names-serverless.md
Comment (florence-crl) [reply src/current/_includes/cockroachcloud/metric-names-serverless.md]: Removed reference to Serverless.

File: src/current/cockroachcloud/custom-metric-chart-page.md
Comment (florence-crl) [reply src/current/cockroachcloud/custom-metric-chart-page.md]: Updated instances to "Metrics". renamed files.

File: src/current/cockroachcloud/custom-metric-chart-page.md
Comment (florence-crl) [reply src/current/cockroachcloud/custom-metric-chart-page.md]: Updated.

Comment (florence-crl) [reply src/current/cockroachcloud/custom-metric-chart-page.md]: Updated.

File: src/current/cockroachcloud/custom-metric-chart-page.md
Comment (florence-crl) [reply src/current/cockroachcloud/custom-metric-chart-page.md]: Updated.

File: src/current/cockroachcloud/custom-metric-chart-page.md
Comment (florence-crl) [reply src/current/cockroachcloud/custom-metric-chart-page.md]: Updated.

File: src/current/cockroachcloud/custom-metric-chart-page.md
Comment (florence-crl) [reply src/current/cockroachcloud/custom-metric-chart-page.md]: I updated it to "Available metrics".

**Review by florence-crl — COMMENTED** [review]
@gemma-shay TFTR! incorporated your suggestions

File: src/current/cockroachcloud/custom-metric-chart-page.md
Comment (florence-crl) [reply src/current/cockroachcloud/custom-metric-chart-page.md]: Updated.

File: src/current/cockroachcloud/custom-metrics-chart-page.md
Comment (THardy98) [inline src/current/cockroachcloud/custom-metrics-chart-page.md:30]: Not sure if here or in the "Units" section within the column/description table makes more sense, but it would be useful to mention that available metric options are filtered by unit. 

Specifically:

Unit selection can be populated 2 ways: 
- by selecting a unit option
- when the first metric has been added - if no unit has been selected yet the metric's unit will populate the unit selection

Once a unit is selected, the available metric options will be filtered to metrics of that unit type (i.e. if the `BYTES` unit is selected, only metrics that are measured in `BYTES` will be available as metric options)

Note that changing the unit will clear the selected metrics.

**Review by THardy98 — COMMENTED** [review]
Nice! Thanks for doing this, left a couple nits and 1 suggestion.

File: src/current/cockroachcloud/custom-metrics-chart-page.md
Comment (THardy98) [reply src/current/cockroachcloud/custom-metrics-chart-page.md:30]: Just remembered this - might be good to mention the `Clear metrics` button as well (which effectively resets the modal to a blank state - no metrics, unit, or aggregators set)

Comment (florence-crl) [reply src/current/cockroachcloud/custom-metrics-chart-page.md:30]: Updated descriptions for both the Metric and Units rows of the columns/description table.

**Review by florence-crl — COMMENTED** [review]
@THardy98 TFTR! I updated with your suggestions.

**Review by THardy98 — COMMENTED** [review]
Left a final nit. Thanks for addressing these so quickly!

File: src/current/_includes/cockroachcloud/ui-custom-metrics-chart-page.html
Comment (florence-crl) [reply src/current/_includes/cockroachcloud/ui-custom-metrics-chart-page.html:22]: updated descriptions in both Metric and Units rows.

File: src/current/v23.1/cockroachdb-feature-availability.md
Comment (gemma-shay) [inline src/current/v23.1/cockroachdb-feature-availability.md:42]:
```suggestion
### Custom Metrics Chart page for CockroachDB {{ site.data.products.serverless }} clusters

The [**Custom Metrics Chart** page]({% link cockroachcloud/custom-metrics-chart-page.md %}) for CockroachDB {{ site.data.products.serverless }} clusters allows you to create custom charts showing the time series data for an available metric or combination of metrics.

```

File: src/current/v23.2/cockroachdb-feature-availability.md
Comment (gemma-shay) [inline src/current/v23.2/cockroachdb-feature-availability.md:42]:
```suggestion
### Custom Metrics Chart page for CockroachDB {{ site.data.products.serverless }} clusters

The [**Custom Metrics Chart** page]({% link cockroachcloud/custom-metrics-chart-page.md %}#) for CockroachDB {{ site.data.products.serverless }} clusters allows you to create custom charts showing the time series data for an available metric or combination of metrics.

```

File: src/current/_includes/cockroachcloud/metric-names-serverless.md
Comment (gemma-shay) [inline src/current/_includes/cockroachcloud/metric-names-serverless.md:20]:
```suggestion
`jobs.row_level_ttl.resume_completed` | Number of `row_level_ttl` jobs which successfully resumed to completion
```
I think job names like this should probably be in code format, applies to other descriptions in this table as well

**Review by gemma-shay — APPROVED** [review]
some last nits but lgtm!

File: src/current/_includes/cockroachcloud/metric-names-serverless.md
Comment (florence-crl) [reply src/current/_includes/cockroachcloud/metric-names-serverless.md:20]: updated.

File: src/current/v23.1/cockroachdb-feature-availability.md
Comment (florence-crl) [reply src/current/v23.1/cockroachdb-feature-availability.md:42]: updated

File: src/current/v23.2/cockroachdb-feature-availability.md
Comment (florence-crl) [reply src/current/v23.2/cockroachdb-feature-availability.md:42]: updated

**Review by florence-crl — COMMENTED** [review]
@gemma-shay thanks for your thoroughness!

**Pull request merged**
